cadc-storage-permissions/src/main/java/org/opencadc/inventory/permissions/Grant.java

```diff
@@ -105,26 +99,22 @@ public Grant(URI artifactURI) {
     
     /**
      * Get the artifactURI to which this grant applies.
-     * @return The artifactURI
+     * @return The artifactURI.
      */
     public URI getArtifactURI() {
         return artifactURI;
     }
-    
+
     /**
-     * Get the read group list.
-     * @return The read groups
+     * Get date after which the Grant is considered expired and should be renewed.
+     * @return The grant expiry date.
      */
-    public List<GroupURI> getReadGroups() {
-        return readGroups;
-    }
-
+    public abstract Date getExpiryDate();
```

Comment:
getExpiryDate() is required but it doesn't need to be abstract; just return the value.

Reply:
Done.